src/components/flat-table/flat-table.style.ts

```diff
@@ -364,6 +364,14 @@ StyledFlatTableWrapper.defaultProps = {
 const StyledFlatTableFooter = styled.div<
   Pick<FlatTableProps, "hasStickyFooter">
 >`
+  & > ${StyledPagerContainer} {
```

Comment:
**comment** (non-blocking): I'm just wondering if we should unit-test these styles, since we were doing that before, and are also still unit-testing the non-table-specific Pager styles.

It's likely not actually necessary since Chromatic should capture any regressions in table pagination styles, so I'm happy to not do it - I just thought I'd mention it in case this was an oversight and you'd rather have the (slight) added confidence of some unit tests here.